Response/Preflight/GraphQl/PreflightRequestHandler.php

```diff
@@ -47,7 +47,7 @@ public function aroundDispatch(GraphQlController $subject, callable $next, Reque
     {
         if ($request instanceof Http && $request->isOptions()) {
             $this->_headerManager->beforeSendResponse($this->_response);
-            $this->_response->setNoCacheHeaders();
+            $this->_response->setPublicHeaders(86400);
```

Comment:
`setPublicHeaders` is setting the cache max age I assume?

Reply:
Correct, it sets the cache age for Varnish, not the browser cache. Browser only respects `Access-control-max-age` whereas varnish determines its cache from the Cache-control max age.